src/django_upgrade/fixers/null_boolean_field.py

```diff
@@ -69,20 +69,21 @@ def visit_Call(
 
 
 def fix_null_boolean_field(tokens: list[Token], i: int, *, node: ast.Call) -> None:
-    j = find(tokens, i, name=OP, src="(")
-    func_args, j = parse_call_args(tokens, j)
+    if "null" not in [keyword.arg for keyword in node.keywords]:
```

Comment:
slight simplification...

```suggestion
    if not any(k.arg == "null" for k in node.keywords):
```